scrapyd/tests/test_utils.py

```diff
@@ -98,6 +99,10 @@ def test_get_spider_list(self):
         self.assertEqual(sorted(spiders), ['spider1', 'spider2'])
 
     def test_get_spider_list_unicode(self):
+        # TODO fix this test on windows
+        # TODO switch to pytest from trial
+        if os.name == 'nt':
+            raise SkipTest
```

Comment:
I assume the PR is still draft?

Reply:
If the plan is to leave these TODOs, then GitHub issues should be created for each of them.

Reply:
fixed here https://github.com/scrapy/scrapyd/pull/412/commits/50e47f26a7682aaefc81737d18b0b332c7d52bfb